### ISSUE TITLE
@next: @twind/preset-tailwind Tree-shaking with preset override problem

### ISSUE THREAD

**Issue by huijiewei** [tw-in-js/twind#329]
I write preset to override preset-tailwind only use nine color, but webpack build result has all color.

```
{
    theme: {
      screens: { mobile: '640px', tablet: '768px', laptop: '1024px', desktop: '1280px' },
      fontFamily: {
        sans: [
          '-apple-system',
          'BlinkMacSystemFont',
          '"Segoe UI"',
          'Helvetica',
          'Arial',
          'sans-serif',
          '"Apple Color Emoji"',
          '"Segoe UI Emoji"',
          '"Segoe UI Symbol"',
        ],
        mono: ['SFMono-Regular', 'Consolas', '"Liberation Mono"', 'Menlo', 'Courier', 'monospace'],
      },
      fontSize: {
        xs: ['0.50rem', '1rem'],
        sm: ['0.75rem', '1.25rem'],
        base: ['0.875rem', '1.5rem'],
        lg: ['1rem', '1.75rem'],
        xl: ['1.5rem', '1.75rem'],
      },
      colors: {
        inherit,
        current,
        transparent,
        black,
        white,
        blue,
        red,
       green,
       yellow,
     },
    }
  }```

**Comment by huijiewei** [comment]
should can custom preset-tailwind  theme

**Comment by sastan** [comment]
I have the idea to provide a way to create a default theme without any default colors to prevent this issue.

**Comment by sastan** [comment]
This is now possible. Here is an example using radix ui colors: https://next.twind-style.pages.dev/preset-radix-ui#with-twindpreset-tailwind

This is not yet released (#421)
